scripts/run-kani.sh

```diff
@@ -151,10 +155,43 @@ get_kani_path() {
     echo "$(realpath "$build_dir/scripts/kani")"
 }
 
-run_kani_command() {
+# Run kani list with JSON format and process with jq to extract harness names and total number of harnesses.
+# Note: The code to extract ALL_HARNESSES is dependent on `kani list --format json` FILE_VERSION 0.1.
+# If FILE_VERSION changes, this logic may need to change as well.
+get_harnesses() {
```

Comment:
How about testing for the expected value of `FILE_VERSION` ?

Reply:
Good idea -- done.